bin/move_it.py

```diff
@@ -645,13 +653,13 @@ def main():
 
     LOGGER.info("Starting up.")
 
-    mask = (pyinotify.IN_CLOSE_WRITE |
+    mask = (pyinotify.IN_MODIFY |
```

Comment:
I'm ok with adding in_modify, but I think in_close_write should be there too.

Reply:
Actually I'm not sure anymore. I think we should wait until the file is closed before we reload it. Otherwise we take the risk of reloading a partially modified file if the editor of the user if saving automatically for example.

Reply:
`IN_CLOSE_WRITE` means that the config will be reloaded everytime the file is opened for viewing and then closed. It literally means "writable file was closed". `IN_MODIFY` means the file was modified, and that only happens when it is saved. Sure, automatical saving can cause issues, but it's either that, reload even after `cat file.yaml`, or no automatic updates. I vote for the implementation I have here.

Reply:
`cat` doesn't open the file in write mode. I think `IN_CLOSE_WRITE` is more appropriate since it won't trigger a reload with a half finished file that is saved. And the `reload_config` function should in principle not restart anything if the file hasn't changed.

Reply:
After some thought, I reverted this event to `IN_CLOSE_WRITE` and adjusted the logging so that it doesn't seem like something was changed if the file was just opened and closed with an editor.